src/components/Subjects/index.js

```diff
@@ -0,0 +1,17 @@
+import React, { useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { fetchSubjects } from "../../store/subject/actions";
+import { selectSubject } from "../../store/subject/selectors";
+import Subject from "../Subject";
+
+export default function ListSubjects() {
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(fetchSubjects());
+  }, [dispatch]);
+
+  const subjects = useSelector(selectSubject);
+
+  return <Subject subjects={subjects} />;
```

Comment:
This component seems unnecessary, it this would be the list we would map over the list of subjects to render !== `Subject` components. But it seems like all this does is fire the `useEffect` to get the data and then `Subject` does all the work. 
Or move the useEffect to `Subject` or move some of the mapping logic to this component